src/coreclr/gc/gc.cpp

```diff
@@ -12093,30 +12335,52 @@ void gc_heap::distribute_free_regions()
         // just to reduce the number of #ifdefs in the code below
         const int i = 0;
 #endif //MULTIPLE_HEAPS
+
         for (int kind = basic_free_region; kind < kind_count; kind++)
         {
-            total_num_free_regions[kind] += hp->free_regions[kind].get_num_free_regions();
+            // If there are regions in free that haven't been used in 20 GCs we always decommit them.
```

Comment:
might want to say `AGE_IN_FREE_TO_DECOMMIT` instead of 20 here.

Reply:
Makes sense.